### PR TITLE
QPPBSR-4161 Add organization schema

### PR DESCRIPTION
**Pull request by seescode**
Adding organization schema to be used on both the bsr client and api.

### CODE REVIEW

File: lib/schema/organization.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export enum GroupSize {
+  OneHundredOrGreater = '100 or More Individual Eligible Clinicians',
+  TwentyFiveNinetyNine = '25 - 99 Individual Eligible Clinicians',
+  TwoTwentyFour = '2 - 24 Individual Eligible Clinicians'
+}
+
+export const OrganizationSchema = Joi.object().keys({
+  name: Joi.string().max(128).regex(Regexes.containsNonWhitespace, 'ContainsNonWhitespace').required(),
+  nickname: Joi.string().max(128).regex(Regexes.containsNonWhitespace, 'ContainsNonWhitespace').allow(null),
+  groupSize: Joi.any().valid(GroupSize.OneHundredOrGreater, GroupSize.TwentyFiveNinetyNine, GroupSize.TwoTwentyFour)
```
Comment (jrmce) [inline lib/schema/organization.ts]: Can we make `groupSize` a string instead of any?

Comment (seescode) [reply lib/schema/organization.ts]: I couldn't find anything like .valid() for a Joi.string().  I could switch it to a regex, but I like the valid() syntax better.

Comment (seescode) [reply lib/schema/organization.ts]: https://github.com/hapijs/joi/blob/master/API.md#anyvalidvalue---aliases-only-equal

Comment (seescode) [reply lib/schema/organization.ts]: Nevermind, it looks like string() inherits from any() so I get all the same functions

**Pull request merged**